maya/mel/assign-pbr-textures.mel

```diff
@@ -0,0 +1,43 @@
+{
+    $sl = {""};
+    $sl = `ls -sl`;
+    
+    if($sl[0]=="")
+    {
+        print "No selection to apply the material to...\n";
+    }
+    else
+    {
+        loadPlugin "C:/Program Files/Autodesk/Maya2017/bin/plug-ins/glslShader.mll";
```

Comment:
Is the absolute path needed? I think you can just pass `glslShader.mll`. Could you try this?